_data/compose-cli/docker_compose_up.yaml

```diff
@@ -73,16 +78,7 @@ options:
   default_value: "false"
   description: 'Detached mode: Run containers in the background'
   deprecated: false
-  experimental: false
-  experimentalcli: false
-  kubernetes: false
-  swarm: false
-- option: environment
-  shorthand: e
```

Comment:
The `-e` / `--environment` option was deprecated/removed? I couldn't find that mentioned in the release notes on https://github.com/docker/compose/releases